<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="keysManagerTab">
      <attribute name="title">
       <string>Keys Manager</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeView" name="allKeysTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImport">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExport">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnInfo">
           <property name="text">
            <string>&amp;Diagnostics</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="knownKeysTab">
      <attribute name="title">
       <string>Known Keys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="knownKeysTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="deleteKnownKey">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="spacerLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="spacerLabel_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ownKeysTab">
      <attribute name="title">
       <string>Own Keys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="accounts"/>
         </item>
         <item>
          <widget class="QPushButton" name="chooseKey">
           <property name="text">
            <string>Select &amp;Key...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ownKeysTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="deleteOwnKey">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="spacerLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="spacerLabel_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="configurationTab">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="encryptionPolicy">
         <property name="title">
          <string>OpenPGP encryption policy</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="alwaysEnabled">
            <property name="text">
             <string>Always enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="enabledByDefault">
            <property name="text">
             <string>Enabled by default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disabledByDefault">
            <property name="text">
             <string>Disabled by default</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpgAgentConfiguration">
         <property name="title">
          <string>gpg-agent configuration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;b&gt;Attention!&lt;/b&gt; This option affects all programs which use gpg-agent in your system!</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Ask password confirmation each</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="pwdExpirationTime">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>60</number>
                 </property>
                 <property name="maximum">
                  <number>2147483647</number>
                 </property>
                 <property name="singleStep">
                  <number>60</number>
                 </property>
                 <property name="value">
                  <number>34560000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>seconds</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openGpgAgentConfig">
              <property name="text">
               <string>Open configuration file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoAssign">
         <property name="text">
          <string>Automatically assign known keys to contacts with signed presences and no assigned keys</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPgpInfoInTooltips">
         <property name="text">
          <string>Show PGP info in tooltips for contacts in roster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoImportPgpKeyFromMessage">
         <property name="text">
          <string>Automatically import PGP key from the message body</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideMessagesWithPgpKeys">
         <property name="text">
          <string>Do not show messages with PGP keys</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>addKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>deleteKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInfo</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>showInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addKey()</slot>
  <slot>deleteKey()</slot>
  <slot>showInfo()</slot>
 </slots>
</ui>
